### DISCUSSION TITLE
RTMP to RTSP

### DISCUSSION THREAD

**Discussion by emmantsia** [bluenviron/mediamtx#294] [Questions]
## Which version are you using?

v0.15.2

## Question

Hello.
I had used an older version before rtmp support.
Until now i used an nginx server for rtmp incoming stream and ffmpeg to send the stream to rtsp for rtsp clients.

As i understand if i stream rtmp to your application it will be directly available to rtsp clients without ffmpeg. Right?
When i don't have any stream running, but i have connected clients is there any option to play a (loop) video until someone will stream? An how? (I think fallback but how a simple video)
And something strange. Is it possible to stream any text with rtsp? (title? caption?) 

Keep the good work! You save many peoples with that tool!

Thank you!

**Comment by aler9** [comment]
Hello, in my opinion, the best option for your use case consists in using OBS Studio (https://obsproject.com/) to generate a live stream with text, fallback videos and anything you want - the problem of the fallback stream is that users can't go back to the original stream if it is back online.

[a comment by an automated account is not shown]
